cmd/juju/application/deploy.go

```diff
@@ -137,11 +137,17 @@ type modelConfigClient struct {
 	*modelconfig.Client
 }
 
+type addCharmClient struct {
```

Comment:
This looks like only used in export_test, so move it there. Is it needed?

Reply:
Was following the pattern that's already introduced in that file with everything being wrapped in a local `struct`. Probably isn't required,